src/default-config-backend.cpp

```diff
@@ -34,37 +77,26 @@ static int handle_config_updated(int fd, uint32_t mask, void *data)
     }
 
     LOGD("Reloading configuration file");
-
     char buf[INOT_BUF_SIZE] __attribute__((aligned(alignof(inotify_event))));
 
     bool should_reload = false;
-    inotify_event *event;
-
-    while (true)
+    if (read_event(fd, buf))
```

Comment:
It seems like the loop here is swapped out. I think I should do this with my other PR as well.